src/ContactCard/ContactCard.css

```diff
@@ -5,8 +5,10 @@
   margin-top: 8%;
   width: 60%;
   height: 50vh;
-  box-shadow: 12px 12px 25px 1px rgba(194, 3, 252, .9),
-  -12px -12px 25px 1px rgba(194, 3, 252, .4);
+  box-shadow: 12px 12px 25px 2px rgba(194, 3, 252, .9),
+  -12px -12px 25px 2px rgba(194, 3, 252, .4);
+  background-color: rgb(0, 0, 0, .6);
```

Comment:
Good use of adding the background color. The lightHouse audit is now 100% on all views.